File: Cmdlets/RemoveCultureAndRegionInfoCommand.vb
Imports System.Management.Automation

<Cmdlet("Remove", "CultureAndRegionInfo")>
Public Class RemoveCultureAndRegionInfoCommand
	Inherits CultureAndRegionInfoCommandBase

	<Parameter(Mandatory:=True)>
	Public Property Name() As String


	Protected Overrides Sub BeginProcessing()
		CultureAndRegionInfoDictionary.Remove(Name)
	End Sub
End Class

File: C#/Cmdlets/Cmdlets/RemoveCultureAndRegionInfoCommand.cs
using System.Management.Automation;

namespace TIKSN.Cmdlets
{
    [Cmdlet("Remove", "CultureAndRegionInfo")]
    public class RemoveCultureAndRegionInfoCommand : CultureAndRegionInfoCommandBase
    {
        [Parameter(Mandatory = true)]
        public string Name { get; set; }

        protected override void BeginProcessing()
        {
            CultureAndRegionInfoDictionary.Remove(Name);
        }
    }
}